setup/fedora.md

```diff
@@ -0,0 +1,193 @@
+# Development Environment Setup on Fedora 28
+
+This is a short guide on how to set up the compiler and interpreter for prototype Xalgorithm Rules on **Fedora 28**. A similar process could be followed on most GNU/Linux distributions or within the Windows Subsystem for Linux. Current interpreter builds (July 2018) use JSON representations of tables, rules, package information and documents. Refer to the [project documentation](https://github.com/xalgorithms/general-documentation) for more information about the working implementation.
+
+Configuring these repositories will prepare all prerequisites for contributing to the project by improving the rule interpreter.
+
+## What is an Xalgorithm Rule?
+
+A **Rule** is a table-oriented expression of a documented computational rule, typically in the domain of commerce or tax. Rules are part of a **Package**, which support the rule with revision history, tables, uniform resource identifier and terms of use.
+
+The [Xalgorithms Alliance](https://www.xalgorithms.org) have laid the foundations for an *Internet of Rules*, where computational rules used by many parties in commerce and government are made accessible, enabling transparency and consistency in the aforementioned applications. Xalgorithms (External, Extensible Algorithms,) will support the automation of taxes, trade fees and algorithmic pricing, along with more ordinary transactions.
+
+Please refer to the [project documentation](https://github.com/xalgorithms/general-documentation) or [Xalgorithms site](https://www.xalgorithms.org) to learn more about the project.
+
+## Installation
+
+After cloning these repositories and installing dependencies, all files that are required to build the compiler, run rules, and contribute to the interpreter will be installed.
+
```

Comment:
These should be references to README in the top-level repo (https://github.com/Xalgorithms/interlibr). Otherwise, we have to keep these component descriptions up to date with those in that file.